.circleci/config.yml

```diff
@@ -16,39 +16,68 @@ jobs:
       - checkout
       - setup_remote_docker
       - run:
-          command: |
-            /scripts/build-image.sh web3f/cla-assistant .
+          command: /scripts/build-image.sh web3f/cla-assistant .
 
-  integrationTests:
+  publishImage:
     docker:
       - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
       - setup_remote_docker
       - run:
-          description: run integration tests
-          command: |
-            /scripts/integration-tests.sh
+          command: /scripts/publish-image.sh web3f/cla-assistant
 
-  publishImage:
+  testChart:
     docker:
       - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
       - setup_remote_docker
       - run:
           command: |
-            /scripts/publish-image.sh web3f/cla-assistant
+            helm lint ./charts/cla-assistant/.
+      - run:
+          command: |
+            helm install budapest-acipenser ./charts/cla-assistant/.
+      - run:
+          command: |
+            helm test budapest-acipenser
 
-  publishChart:
+
+  integrationTests:
     docker:
       - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
+      - setup_remote_docker
+      - run:
```

Comment:
Given that we are using the integration tests local script maybe we could move this run command to it? So that all the tests that require a running cluster are in the same place.